sitecore-xp0-sxa-horizon/docker/build/cm/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# escape=`
+
+ARG BASE_IMAGE
+ARG SXA_IMAGE
+ARG SPE_IMAGE
+ARG HRZ_IMAGE
+ARG TOOLING_IMAGE
+
+FROM ${TOOLING_IMAGE} as tooling
+FROM ${SPE_IMAGE} as spe
+FROM ${SXA_IMAGE} as sxa
+FROM ${HRZ_IMAGE} as horizon
+FROM ${BASE_IMAGE}
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Copy development tools and entrypoint
+COPY --from=tooling \tools\ \tools\
+
+WORKDIR C:\inetpub\wwwroot
+
+#Enable WebSockets
+RUN powershell -Command Add-WindowsFeature Web-WebSockets
```

Comment:
You don't need "powershell -Command" here since we set the SHELL to PowerShell on Line15